### DISCUSSION TITLE
I deleted the web interfrace folder somehow

### DISCUSSION THREAD

**Discussion by bakissation** [OpenPrinting/cups#817] [Q&A]
Hello,

So CUPS is working completely fine, but I lost the WI and the web folder completely (idk how, but I belive I did it, I trust my ability to do such mistakes) , I don't want to reset my system, any suggestion on how to restore the GUI it was really useful?

**Comment by dkosovic** [comment]
I'm guessing the CUPS Web Interface isn't in a disabled state, so enabling it with `sudo cupsctl WebInterface=yes` won't have any effect, because if you go to http://localhost:631 and it were disabled, it will tell you to enable the web interface using that `cupsctl` command

Assuming you did delete the actual CUPS web folder which has the static content, it is typically either `/usr/share/cups/www/` or `/usr/share/doc/doc-root/` (or whatever the equivalent is for your Linux distro). Most Linux distros allow you to re-install packages without replacing config files.

Without knowing your Linux distro, it is hard to give advice.

On Fedora and RHEL you can do the following to reinstall the **cups** package which has the `/usr/share/cups/www/` static web content:

```
sudo dnf reinstall cups
```

On Debian or Ubuntu, I believe it would be the **cups-server-common** package that has `/usr/share/doc/doc-root` static web content:

```
sudo apt  install --reinstall cups-server-common
```
